src/app/components/Typography/Typography.tsx

```diff
@@ -33,5 +35,9 @@ export default function Typography({
       return <p className={styleMap[size]}>{children}</p>;
     case 'ps':
       return <p className={styleMap[size]}>{children}</p>;
+    case 'xl':
+      return <h1 className={styleMap[size]}>{children}</h1>;
+    case 'pp':
+      return <p className={styleMap[size]}>{children}</p>;
```

Comment:
I like it, but due to the fact that you've added another `h1` and `p` element, I'd suggest switching to a map. Pun intended